website/static/js/meetingsAndConferencesPlugin.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Meetings and Conferences
+ */
+
+var $ = require('jquery');
+var m = require('mithril');
+var $osf = require('js/osfHelpers');
+
+// CSS
+require('css/quick-project-search-plugin.css');
+
+// XHR config for apiserver connection
+var xhrconfig = function(xhr) {
+    xhr.withCredentials = true;
+};
+
+
+var meetingsAndConferences = {
+    controller: function() {
+        var self = this;
+        self.findAMeeting = function(){
+            location.href = '/meetings/';
+        };
+    },
+    view: function(ctrl) {
+        function findAMeetingButton() {
+             return m('button', {type:'button', 'class':'btn btn-default btn-block m-v-xl', onclick: function(){
+                ctrl.findAMeeting();
+            }}, 'Find a Meeting');
+
+        }
+
+        function createAMeetingButton() {
+             return m('button', {type:'button', 'class':'btn btn-default btn-block m-v-xl', onclick: function(){
+                ctrl.findAMeeting();
+            }}, 'Create a Meeting');
+        }
+        return m('div', {'class': 'container conferences-and-meetings m-v-sm p-v-sm'}, [
```

Comment:
They work somewhat like css selectors but without spaces so a lot of combinations are possible like 
```
m('span.class#id[type="input"]')...
```